### PR TITLE
feat: add kdcproxy tasks

### LINKED ISSUE #14: Add kdcproxy

### PR DESCRIPTION
**Pull request by gboutry**
#### Which issue(s) this PR fixes:

Fixes #14

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: topology.ini [hunk 1]
```diff
@@ -4,5 +4,8 @@ master1
 [kdc:children]
 master1
 
+[kdcproxy:children]
+master3
```
Comment (rpignolet) [inline topology.ini:8]: Better on `edge` like Knox no ?

**Review by rpignolet — APPROVED** [review]

**Comment by gboutry** [comment]
force push is rebase on master and solving conflicts

File: roles/ldap_kerberos/defaults/main.yml [hunk 2]
```diff
@@ -86,4 +86,33 @@ kerberos_kadmin_admin_template: "*/admin@{{ realm }}"
...
+  {{
+    groups['kdcproxy']
+    | map('regex_replace', '^(.*)$', 'https://\1.' + domain + '/KdcProxy')
+    | list
+  }}
+
+certificates_path: /etc/ssl/certs
+certificates_ca_filename: tdp_ca
+
+httpd_ca_root: "{{ certificates_path }}/{{ certificates_ca_filename }}.crt"
```
Comment (leopaul36) [inline roles/ldap_kerberos/defaults/main.yml:106]: Is this variable used at some point?

Comment (gboutry) [reply roles/ldap_kerberos/defaults/main.yml:106]: yes

Comment (gboutry) [reply roles/ldap_kerberos/defaults/main.yml:106]: In ssl-httpd.conf.j2, it's templated inside

Comment (gboutry) [reply roles/ldap_kerberos/defaults/main.yml:106]: Ok, it was missing, I put it back

**Review by leopaul36 — APPROVED** [review]

**Pull request merged**